src/components/pages/ProfileList/RenderProfileListPage.js

```diff
@@ -26,39 +27,14 @@ function RenderProfileListPage(props) {
   }
 
   return (
-    <div>
+    <div className="profile-container">
       <p>
         <Link to="/">Home</Link>
       </p>
       <SearchForm value={searched} handleChange={handleChange} />
       <div className="profile-list">
         {filtered.map(item => (
-          <Card
-            className="profile-card spin"
-            key={item.id}
-            cover={
-              <img
-                className="card-image"
-                src={item.avatarUrl}
-                alt={item.name}
-              />
-            }
-          >
-            <Meta
-              className="card-data"
-              title={item.name}
-              description={'user or groomer'}
-            ></Meta>
-            <br />
-            Rating:
-            <br />
-            <Rate
-              character={<ScissorOutlined />}
-              style={{ 'margin-top': '5%', color: 'black' }}
-              disabled
-              defaultValue={5}
-            />
-          </Card>
+          <ProfileCard item={item} />
```

Comment:
Nice job simplifying this so that it's all contained in a profile card component that can be reused.